pytoda/datasets/drug_affinity_dataset.py

```diff
@@ -19,6 +22,7 @@ def __init__(
         drug_affinity_filepath: str,
         smi_filepath: str,
         protein_filepath: str,
+        column_names: Tuple[str] = ['label', 'ligand_name', 'sequence_id'],
```

Comment:
Since we are free to define the order here, I would use the "natural" one as they appear in our files. Usually a label/class column comes last, so it's not only us. This would make the test a little more readable by getting rid of `np.roll`
```suggestion
        column_names: Tuple[str] = ['ligand_name', 'sequence_id', 'label'],
```